docs/asset-store/assets/example-openapi-service.yaml

```diff
@@ -0,0 +1,128 @@
+openapi: 3.0.0
+info:
+  description: This example shows API specification of a service that exposes endpoints for all Asset Store webhooks. It contains `/convert`, `/validate`, and `/extract` endpoints.
+  title: Example webhook service
+  contact:
+    name: Kyma project authors
+    url: http://slack.kyma-project.io
+    email: kyma-sig-core@googlegroups.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+  version: 0.0.1
+paths:
+  /convert:
+    post:
+      description: Represents the `mutation webhook`.
+      tags:
+        - convert
+      summary: Converts input files.
+      operationId: convertReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                parameters:
+                  description: Additional parameters for the service
+                  type: string
+                content:
+                  description: File to convert
+                  type: string
+                  format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successConversionResponse"
+        "304":
+          $ref: "#/components/responses/notModified"
+  /validate:
+    post:
+      description: Represents the `validation webhook`.
+      tags:
+        - validate
+      summary: Validates input files.
+      operationId: validateReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                parameters:
+                  description: Additional parameters for the service
+                  type: string
+                content:
+                  description: File to validate
+                  type: string
+                  format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successValidationResponse"
+        "422":
+          $ref: "#/components/responses/unprocessableEntity"
+  /extract:
+    post:
+      description: Represents the `metadata webhook`.
+      tags:
+        - extract
+      summary: Extracts metadata from input files.
+      operationId: extractReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              additionalProperties:
+                type: string
+                format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successResponse"
+servers:
+  - url: http://localhost:3000/v1
+components:
+  responses:
+    successConversionResponse:
+      description: Successful response with the new file content
```

Comment:
```suggestion
      description: A successful response with the new file content
```

Reply:
As discussed, we are omitting the articles at the beginning of response descriptions.